backend/test_observer/controllers/test_executions/models.py

```diff
@@ -103,6 +104,21 @@ class TestExecutionsPatchRequest(BaseModel):
     c3_link: HttpUrl | None = None
     ci_link: HttpUrl | None = None
     status: TestExecutionStatus | None = None
+    review_status: list[TestExecutionReviewDecision] | None = None
+    review_comment: str | None = None
+
+    @model_validator(mode="after")
+    def validate_review_status(self) -> "TestExecutionsPatchRequest":
+        if self.review_status is None:
+            return self
+
+        if len(self.review_status) <= 1:
+            return self
+
+        for review_status in self.review_status:
+            if review_status in (TestExecutionReviewDecision.REJECTED,):
+                raise ValueError("Test execution can either be rejected or approved")
+        return self
```

Comment:
this can be a [`field_validator`](https://docs.pydantic.dev/latest/concepts/validators/#field-validators) as it's only one field.